src/ovp_pipeline/commands/_ui_renderers.py

```
@@ -1797,12 +1849,17 @@ def _render_note_page(
             requested_pack=requested_pack,
         )
+        # M22 BL-093: prev/next pivot when this is a daily digest
+        # so operators can step through history without bouncing
+        # back to the /digests list every time.
+        digest_nav_html = _render_digest_neighbour_nav(vault_dir, relative_path)
         return _layout(
             f"Markdown Note: {relative_path}",
             (
                 "<h1>Markdown Note</h1>"
                 f"<p class='muted'>{escape(relative_path)}</p>"
                 + f"<div class='entry-actions'>{ask_button}</div>"
```

Ruff (0.15.12)

[warning] Explicitly concatenated string should be implicitly concatenated

Remove redundant '+' operator to implicitly concatenate

(ISC003)